yew/src/virtual_dom/vtag.rs

```diff
@@ -318,6 +318,24 @@ impl VTag {
             }
         }
 
+        if let Some(button) = {
+            // TODO: add std_web after https://github.com/koute/stdweb/issues/395 will be approved
+            // Check this out: https://github.com/yewstack/yew/pull/1033/commits/4b4e958bb1ccac0524eb20f63f06ae394c20553d
+            cfg_match! {
+                feature = "web_sys" => element.dyn_ref::<HtmlButtonElement>(),
```

Comment:
@jstarry  travis-ci complains about unsuported configuration: [here](https://travis-ci.com/github/yewstack/yew/jobs/323468442#L691)

Reply:
@captain-yossarian you need to make sure your code works for `stdweb`:
```sh
(cd yew-stdweb && cargo check)
```